lib/internal/streams/readable.js

```diff
@@ -1568,6 +1572,49 @@ ObjectDefineProperties(ReadableState.prototype, {
       }
     },
   },
+
+  // Legacy compat
```

Comment:
I would add a pending deprecation warning and a runtime deprecation in v22.

Reply:
Agree